build.sh

```diff
@@ -18,7 +18,7 @@ EOF
 	done
 }
 
-check_for_application lex yacc autoheader aclocal automake autoconf
```

Comment:
@octo are you sure? last time I checked collectd didn't build with other yacc implementations.